{"name": "test_is_prime_1_5", "status": "failed", "statusDetails": {"message": "AssertionError: True is not false", "trace": "self = <testcase.test_ddt.TestMyFunc testMethod=test_is_prime_1_5>, a = 5\n\n    @data(5, 7,8,-2,0,1)\n    def test_is_prime(self, a):    #测试用例以test_开头\n        ''' test method is_prime(number) '''\n        print(\"is_prime\")\n>       self.assertFalse(is_prime(a))    #不适合多种断言测试\nE       AssertionError: True is not false\n\ntestcase\\test_ddt.py:37: AssertionError"}, "description": " test method is_prime(number) ", "attachments": [{"name": "stdout", "source": "9f2963dd-9084-4e2a-b2a5-7fd7f7b75d6e-attachment.txt", "type": "text/plain"}], "start": 1572531711867, "stop": 1572531711867, "uuid": "723035a3-1e5f-4298-924e-83edccce6915", "historyId": "a3a95497fd1a3db26595bd0824b7969f", "testCaseId": "19e43ae6478f13cd2fe461e7a72d5cce", "fullName": "testcase.test_ddt.TestMyFunc#test_is_prime_1_5", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_ddt"}, {"name": "subSuite", "value": "TestMyFunc"}, {"name": "host", "value": "LeeYatyi"}, {"name": "thread", "value": "12480-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_ddt"}]}